### PR TITLE
fix(headers): Make questions more visible

### PR DESCRIPTION
**Pull request by vzpgb**

### CODE REVIEW

File: content/resources/faq/_index.md [hunk 1]
```diff
@@ -36,14 +36,14 @@ can find more information in the
...
-access it.**
+<h4>User.yaml contains information about program/project, but I still can not
+access it.</h4>
 
 Please ensure you are using `dbgap_accession_number` value as an identifier for
 the `auth_id` field, not program's `name` or project's `code`.
 
 
-**I see a warning about my connection is not being private.**
+<h4>I see a warning about my connection is not being private.</h4>
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: I'd remove "is"

File: content/resources/faq/_index.md [hunk 2]
```diff
@@ -52,16 +52,16 @@ Your browser can not verify this certificate. Once you install a trusted SSL
...
 
 To configure Compose Services with your domain, please change the `BASE_URL`
 field in the `fence_config.yaml` file, and the `hostname` field in the
 `peregrine.json` and `sheepdog.json` files.
 
 
-**I'm trying to upload files with gen3-client, but receive an error 
-`x509: certificate signed by unknown authority`.**
+<h4>I'm trying to upload files with gen3-client, but receive an error 
+`x509: certificate signed by unknown authority`.</h4>
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: I'd change "receive" to "received"

File: content/resources/faq/_index.md [hunk 3]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
-**I updated Fence configuration but changes don't appear?**
+<h4>I updated Fence configuration but changes don't appear?</h4>
 
 After updating the Fence configuration, run `kubectl delete secret
 fence-config`, `gen3 kube-setup-fence` and roll Fence for configuration changes
 to take effect with `gen3 roll fence`.
 
 
-**GraphiQL on Query page form throwing errors.**
+<h4>GraphiQL on Query page form throwing errors.</h4>
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: Is it "GraphiQL" or "GraphQL"?

File: content/resources/faq/_index.md [hunk 4]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
 First, restart Peregrine `gen3 roll peregrine` and Sheepdog `gen3 roll
 sheepdog`.  After these two services restarted, please portal with `gen3 roll
 portal`.  Please note, if you are making significant changes to the dictionary,
 you will also need to update `portal_app` in the manifest.
 
 
-**I branched out windmill portal, should I update manifest with a new version of
-portal?**
+<h4>I branched out windmill portal, should I update manifest with a new version of
+portal?</h4>
```
Comment (hugheslaron) [inline content/resources/faq/_index.md]: I'd recommend putting "the" between "out" and "windmill".  And capitalize "Windmill"

File: content/resources/faq/_index.md [hunk 5]
```diff
@@ -52,16 +52,16 @@ Your browser can not verify this certificate. Once you install a trusted SSL
 certificate instead of generated by our script, this warning will go away.
 
 
-**I want to run Compose Services on the server, how to configure
- it.**
+<h4>I want to run Compose Services on the server, how to configure
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>I want to run Compose Services on the server, how do I configure
```

File: content/resources/faq/_index.md [hunk 6]
```diff
@@ -52,16 +52,16 @@ Your browser can not verify this certificate. Once you install a trusted SSL
...
+ it.</h4>
 
 To configure Compose Services with your domain, please change the `BASE_URL`
 field in the `fence_config.yaml` file, and the `hostname` field in the
 `peregrine.json` and `sheepdog.json` files.
 
 
-**I'm trying to upload files with gen3-client, but receive an error 
-`x509: certificate signed by unknown authority`.**
+<h4>I'm trying to upload files with gen3-client, but receive an error
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>I'm trying to upload files with the gen3-client, but I receive the error 
```

File: content/resources/faq/_index.md [hunk 7]
```diff
@@ -81,16 +81,16 @@ later when deploying kubernetes, you will be able to point the location of your
 permissions file.
 
 
-**I modified `user.yaml` file. How to update Arborist?**
+<h4>I modified `user.yaml` file. How to update Arborist?</h4>
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>I modified the `user.yaml` file. How to update Arborist?</h4>
```

File: content/resources/faq/_index.md [hunk 8]
```diff
@@ -81,16 +81,16 @@ later when deploying kubernetes, you will be able to point the location of your
...
 
 If you run `gen3 job run usersync` changes should be picked up automatically.
 If you don't want to run that command every time you make changes to your
 `user.yaml`, you can deploy the cronjob `gen3 job run usersync-cronjob` and
 kubernetes will check for changes every hour and update services accordingly.
 
 
-**How to change the string "Generic Data Commons" in webpages of my Cloud
-Automation?**
+<h4>How to change the string "Generic Data Commons" in webpages of my Cloud
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>How do I change the title of my webpage from the default "Generic Data Commons"?
```

File: content/resources/faq/_index.md [hunk 9]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
 title at your choice. Then reroll portal with `gen3 kube-setup-portal`.
 
 
-**I updated Fence configuration but changes don't appear?**
+<h4>I updated Fence configuration but changes don't appear?</h4>
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>I updated the Fence configuration. Why haven't the changes taken effect?</h4>
```

File: content/resources/faq/_index.md (hunk 1 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in 4b5ddf4

File: content/resources/faq/_index.md [hunk 10]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
 
 After updating the Fence configuration, run `kubectl delete secret
 fence-config`, `gen3 kube-setup-fence` and roll Fence for configuration changes
 to take effect with `gen3 roll fence`.
 
 
-**GraphiQL on Query page form throwing errors.**
+<h4>GraphiQL on Query page form throwing errors.</h4>
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>I receive errors when running queries on the GraphiQL "Query" page, e.g., Cannot query field "field" on type "Root".</h4>
```

File: content/resources/faq/_index.md (hunk 2 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in 4b5ddf4

File: content/resources/faq/_index.md [hunk 11]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
 
 
-**Arborist and Sheepdog don't work as expected, what should I check?**
+<h4>Arborist and Sheepdog don't work as expected, what should I check?</h4>
 
 `kubectl logs <name of pod>` can show you corresponding errors in the logs.
 
 
-**I uploaded the new dictionary, what service should I restart?**
+<h4>I uploaded the new dictionary, what service should I restart?</h4>
```
Comment (cgmeyer) [inline content/resources/faq/_index.md]:
```suggestion
<h4>I've updated the data dictionary, what services should I restart for the change to take effect?</h4>
```

File: content/resources/faq/_index.md (hunk 4 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in 4b5ddf4

File: content/resources/faq/_index.md (hunk 3 above)
Comment (abgeorge7) [reply content/resources/faq/_index.md]: GraphiQL is the UI, GraphQL is the API :P

File: content/resources/faq/_index.md [hunk 12]
```diff
@@ -12,21 +12,21 @@ menuname: faqMenu
...
+<h4>I installed Compose Services, how do I create a program and a project?</h4>
 
 Please refer to corresponding
 [Gen3 Documentation](https://gen3.org/resources/operator/#5-programs-and-projects)
 to create your first program and project.
 
 
-**I created a program, but I see `NotFound` error when I try to create a
-project within my program.**
+<h4>I created a program, but I see `NotFound` error when I try to create a
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "... a `NotFound` error..."

File: content/resources/faq/_index.md (hunk 3 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: In the Windmill it is GraphiQL

File: content/resources/faq/_index.md [hunk 13]
```diff
@@ -36,14 +36,14 @@ can find more information in the
 documentation.
 
 
-**User.yaml contains information about program/project, but I still can not
-access it.**
+<h4>User.yaml contains information about program/project, but I still can not
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "can not" should be "cannot"

File: content/resources/faq/_index.md [hunk 14]
```diff
@@ -52,16 +52,16 @@ Your browser can not verify this certificate. Once you install a trusted SSL
 certificate instead of generated by our script, this warning will go away.
 
 
-**I want to run Compose Services on the server, how to configure
- it.**
+<h4>I want to run Compose Services on the server, how do I configure
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: add a question mark to the end

File: content/resources/faq/_index.md [hunk 15]
```diff
@@ -81,16 +81,16 @@ later when deploying kubernetes, you will be able to point the location of your
 permissions file.
 
 
-**I modified `user.yaml` file. How to update Arborist?**
+<h4>I modified the `user.yaml` file. How to update Arborist?</h4>
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "How do I update Arborist?"

File: content/resources/faq/_index.md [hunk 16]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
 
-**GraphiQL on Query page form throwing errors.**
+<h4>I receive errors when running queries on the GraphiQL "Query" page, e.g., Cannot query field "field" on type "Root".</h4>
 
 You may need to hit the `Switch to Graph Model` button in the upper right
 corner.
 
 
-**How to run SNS service?**
+<h4>How to run SNS service?</h4>
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "How do I run SNS services?"

File: content/resources/faq/_index.md [hunk 17]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
 
 First, restart Peregrine `gen3 roll peregrine` and Sheepdog `gen3 roll
 sheepdog`.  After these two services restarted, please portal with `gen3 roll
 portal`.  Please note, if you are making significant changes to the dictionary,
 you will also need to update `portal_app` in the manifest.
 
 
-**I branched out windmill portal, should I update manifest with a new version of
-portal?**
+<h4>I branched out the Windmill portal, should I update manifest with a new
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: I don't 100% understand what "branched out" mean here

File: content/resources/faq/_index.md [hunk 18]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
-**How can I update the APP for the portal?**
+<h4>How can I update the APP for the portal?</h4>
 
 You can set the `global.portal_app` setting in your manifest to the name of your
 windmill configuration file and run `gen3 kube-setup-portal`.
 
 
-**What I need to restart if I updated Fence configuration with OAuth
-credentials?**
+<h4>What I need to restart if I updated Fence configuration with OAuth
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "What do I need to restart..."

File: content/resources/faq/_index.md [hunk 19]
```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
...
+credentials?</h4>
 
 Please run `kubectl delete secret fence-config` and `gen3 kube-setup-fence`.  If
 you also updated the `user.yaml` in the folder `apis_configs`, you need to run
 `gen3 job run useryaml-job`.
 
 
-**I'm storing `user.yaml` in S3 (not locally) and updated it, how to ensure
-changes take effect?**
+<h4>I'm storing `user.yaml` in S3 (not locally) and updated it, how to ensure
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "how do I ensure"

File: content/resources/faq/_index.md [hunk 20]
```diff
@@ -175,17 +175,17 @@ Configuration](https://github.com/uc-cdis/data-portal/pull/505) for the Guppy.
 
 ## Data Dictionary
 
-**I'm customizing data schema, where I can find information about the data
-dictionary customization, variables' definition, and files formatting?**
+<h4>I'm customizing data schema, where I can find information about the data
```
Comment (abgeorge7) [inline content/resources/faq/_index.md]: "I'm customizing the data schema..."

File: content/resources/faq/_index.md (hunk 17 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Created branch from the repo, forked.

File: content/resources/faq/_index.md (hunk 14 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 12 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 16 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 14 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 13 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 18 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 19 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

File: content/resources/faq/_index.md (hunk 20 above)
Comment (vzpgb) [reply content/resources/faq/_index.md]: Fixed in ca3cbb7

**Pull request merged**